.env.local.example

```diff
@@ -1,2 +1,4 @@
 VITE_LOCAL_HOSTNAME=http://127.0.0.1
-VITE_PROXY_URL=http://localhost:5000
+VITE_PROXY_URL_REGEN_MAINNET=http://api.registry.regen.network
+VITE_PROXY_URL_REGEN_TESTNET=http://api-staging.registry.regen.network
+VITE_INDEXER_GRAPHQL_API=http://localhost:5000/indexer/graphql
```

Comment:
```suggestion
VITE_INDEXER_GRAPHQL_API=http://localhost:5000/indexer/v1/graphql
```

Reply:
Opened a pr so we can add these to the dev branch as well: https://github.com/regen-network/groups-ui/pull/138